recipes-bsp/grub/grub-efi/grub.cfg

```diff
@@ -0,0 +1,15 @@
+serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
+terminal_output console serial
+terminal_input console serial
+default=boot
+timeout=5
+
+menuentry 'boot'{
+linux /bzImage-initramfs-genericx86-64.bin rootwait root=LABEL=root rootfstype=ext4 console=ttyS0,115200 earlyprintk=serial,ttyS0,115200
+}
+
+menuentry 'skl-boot'{
+slaunch skinit
+slaunch_module /skl.bin
```

Comment:
Done in [6e61173](https://github.com/3mdeb/meta-trenchboot/pull/28/commits/6e61173e2e386ca42b5f71a0f27a5fdd76afc58e). After @macpijan moved `grub.cfg` to wic I'm not sure if it's possible to autogenerate those versions (they shouldn't change without changing URL in recipe)

Reply:
Do not assume that Intel won't change it without changing package version, it did so many times in the past. I assume it will fail due to different hash in that case, right?

Reply:
Yes, if they change anything in zip then build will fail and we would need to manually change hash

Reply:
Good, no surprises here.